pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -1371,26 +1208,15 @@ fn can_verify_restriction_with_primary_issuance_agent() {
 }
 
 fn can_verify_restriction_with_primary_issuance_agent_we() {
-    let owner = AccountKeyring::Alice.public();
-    let owner_origin = Origin::signed(owner);
-    let owner_id = register_keyring_account(AccountKeyring::Alice).unwrap();
+    let owner = User::new(AccountKeyring::Alice);
     let issuer = AccountKeyring::Bob.public();
     let issuer_id = register_keyring_account(AccountKeyring::Bob).unwrap();
     let random_guy_id = register_keyring_account(AccountKeyring::Charlie).unwrap();
```

Comment:
```suggestion
    let issuer = User::new(AccountKeyring::Bob);
    let other = User::new(AccountKeyring::Charlie);
```

Reply:
I was thinking of doing the same improves to the other accounts as I did with Alice.  But didn't want to do too much.

I will make these changes for all of the accounts.